config/routes.rb

```diff
@@ -14,6 +14,7 @@
   get '/dashboards' => 'dashboards#index'
   post '/dashboards/build' => 'dashboards#build', as: :dashboard_build
   get '/dashboards/export/:csv_type' => 'dashboards#export', as: :dashboard_export, defaults: { format: 'csv' }
```

Comment:
Given our new URL for `institutions`, I was wondering what would happen if you now visit `/dashboards/export/institutions`?  Will it generate an error?

I think you might need to alter `dashboards_controller#csv_model`.
Maybe that method should raise `ArgumentError` and specify in the message that you must provide a version number for `institutions`

Reply:
There isn't any UI that would hit `/dashboards/export/institutions`, do you think we have to cover that case?

(but this is what you would get even if you tried to access it directly)

![image](https://user-images.githubusercontent.com/41960449/61664864-b3eee200-aca1-11e9-98e9-423b3198083d.png)

Reply:
Ah, I see the old URL was `/dashboards/export/Institution`, not `/dashboards/export/institutions`.
What happens if you hit `/dashboards/export/Institution`?
It is a `GET` URL, so it's possible (if unlikely) that someone could type it in or have it in the browser history or have it bookmarked and visit it that way.

Reply:
I removed the hardcoded rule that let Institution be exported without being in the allowed table list.